### ISSUE TITLE
Uploading Photos

### ISSUE THREAD

**Issue by maxschram** [mdhighfi/Paintr#3]
Are you planning on doing this in phase 2?

**Comment by mdhighfi** [comment]
I guess it will be hard to test the views from phase 1 without any photos from phase 2... is that the issue?

**Comment by maxschram** [comment]
Yeah that's what I'm thinking.It might make it easier if you lumped the photo stuff into one phase.
